build.gradle

```diff
@@ -368,3 +383,19 @@ if (CacheRedirector.enabled) {
         nodeJsExtension.nodeDownloadBaseUrl = CacheRedirector.maybeRedirect(nodeJsExtension.nodeDownloadBaseUrl)
     }
 }
+
+// This is bootstrap atomicFU version with wasm support, remove it as regular atomic-fu will be published
+allprojects {
+    configurations.all {
+        resolutionStrategy.eachDependency { DependencyResolveDetails details ->
+            if (details.requested.name == 'atomicfu-wasm') {
```

Comment:
Is it some workaround for the target resolution?
I'm a bit puzzled why just bumping version from `0.21.0` to `0.21.0-wasm0` is not enough

Reply:
I rather don't get it: all targets use `0.21.0-wasm0` already because of https://github.com/Kotlin/kotlinx.coroutines/blob/45aa4143a2c843220138c910707ee987ebf559af/build.gradle#L65 thus I do not understand the reason behind this `configuration` block.

Regarding the general question -- it's okay to use `0.21.0-wasm0` version everywhere. We are not going to release coroutines before atomicfu, thus is quite safe

Reply:
Fixed